capstone/helpers/image_processing.py

```diff
@@ -0,0 +1,111 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import skimage.io as io
+import skimage.transform as transform
+import skimage.exposure as exposure
+import glob
+import os
+
+
+def print_columns(data, cols):
+    """
+    Splits a list into sublists and prints it out in columns
```

Comment:
Sorry for not mentioning this earlier, but based on PEP8 comments are recommended to be written in imperative.